vizro-core/docs/pages/user_guides/container.md

```diff
@@ -164,3 +163,8 @@ vm.Container(
     ]
 )
 ```
+
```

Comment:
I think this new content gets a bit lost at the bottom of the page and it's not only relevant for nested containers so doesn't really belong here. Please could you put it somewhere near the top of the page instead, maybe in a note callout?

Reply:
I've put them in a note callout now above the section of  "When to use containers"